backend/Modules/Users/HitMeApp.Users/Exceptions/PasswordNotValidException.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace HitMeApp.Users.Exceptions
+{
+    internal class PasswordNotValidException : Exception
```

Comment:
I think `InvalidPasswordException` feels like a better name, but treat it optionally :)

Reply:
Done